vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt

```diff
@@ -1011,4 +1007,21 @@ class VectorPreferences @Inject constructor(private val context: Context) {
             putInt(TAKE_PHOTO_VIDEO_MODE, mode)
         }
     }
+
+    // Layout
+
+    fun getLayoutMode(): LayoutMode {
+        return defaultPrefs.getString(SETTINGS_LAYOUT_MODE, LayoutMode.PRODUCTIVITY.name).let {
```

Comment:
Default is PRODUCTIVITY? I would say SIMPLE would be better, or maybe reuse the setting `SETTINGS_LABS_UNREAD_NOTIFICATIONS_AS_TAB`, if true -> PRODUCTIVITY